cmd/cassandra-reader/main.go

```diff
@@ -34,48 +34,48 @@ import (
 const (
 	sep = ","
 
-	defLogLevel      = "error"
-	defPort          = "8180"
-	defCluster       = "127.0.0.1"
-	defKeyspace      = "mainflux"
-	defDBUsername    = ""
-	defDBPassword    = ""
-	defDBPort        = "9042"
-	defThingsURL     = "localhost:8181"
-	defClientTLS     = "false"
-	defCACerts       = ""
-	defServerCert    = ""
-	defServerKey     = ""
-	defJaegerURL     = ""
-	defThingsTimeout = "1" // in seconds
-
-	envLogLevel      = "MF_CASSANDRA_READER_LOG_LEVEL"
-	envPort          = "MF_CASSANDRA_READER_PORT"
-	envCluster       = "MF_CASSANDRA_READER_DB_CLUSTER"
-	envKeyspace      = "MF_CASSANDRA_READER_DB_KEYSPACE"
-	envDBUsername    = "MF_CASSANDRA_READER_DB_USERNAME"
-	envDBPassword    = "MF_CASSANDRA_READER_DB_PASSWORD"
-	envDBPort        = "MF_CASSANDRA_READER_DB_PORT"
-	envThingsURL     = "MF_THINGS_URL"
-	envClientTLS     = "MF_CASSANDRA_READER_CLIENT_TLS"
-	envCACerts       = "MF_CASSANDRA_READER_CA_CERTS"
-	envServerCert    = "MF_CASSANDRA_READER_SERVER_CERT"
-	envServerKey     = "MF_CASSANDRA_READER_SERVER_KEY"
-	envJaegerURL     = "MF_JAEGER_URL"
-	envThingsTimeout = "MF_CASSANDRA_READER_THINGS_TIMEOUT"
+	defLogLevel          = "error"
+	defPort              = "8180"
+	defCluster           = "127.0.0.1"
+	defKeyspace          = "messages"
+	defDBUser            = "mainflux"
+	defDBPass            = "mainflux"
+	defDBPort            = "9042"
+	defClientTLS         = "false"
+	defCACerts           = ""
+	defServerCert        = ""
+	defServerKey         = ""
+	defJaegerURL         = ""
+	defThingsAuthURL     = "localhost:8181"
+	defThingsAuthTimeout = "1" // in seconds
+
+	envLogLevel          = "MF_CASSANDRA_READER_LOG_LEVEL"
+	envPort              = "MF_CASSANDRA_READER_PORT"
+	envCluster           = "MF_CASSANDRA_READER_DB_CLUSTER"
+	envKeyspace          = "MF_CASSANDRA_READER_DB_KEYSPACE"
+	envDBUser            = "MF_CASSANDRA_READER_DB_USERNAME"
```

Comment:
Please change to `MF_CASANDRA_READER_DB_USER` and `MF_CASANDRA_READER_DB_PASS`. Also, please don't forget to change README, `.env`, and compose.

Reply:
Should I rename also `MF_CASSANDRA_WRITER_DB_KEYSPACE` -> `MF_CASSANDRA_WRITER_DB_NAME`

Reply:
I would say no. I'm afraid it would be misleading and would only confuse the end-user.